test/data.jl

```diff
@@ -184,11 +184,14 @@ end
     df = DataFrame(rand(3,4))
     for fun in (dropmissing, unique)
         @test fun(df) isa DataFrame
+        @inferred fun(df)
```

Comment:
FWIW, I think you can put `@inferred` anywhere in a standard `@test` line.

Reply:
I thought it is better to keep it separate, just to make sure it does not get deleted unintentionally (along your earlier comment that we should be explicit about it).